css/index.css

```diff
@@ -1,3 +1,434 @@
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+body {
+	display: flex;
+	justify-content: center;
+}
+
+.container {
+	width: 880px;
+}
+
+header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-top: 28px;
+}
+
+header nav {
+	display: flex;
+	justify-content: space-between;
+  	width: 690px;
+  	margin-left: 72px;
+}
+
+header nav a {
+  display: inline-block;
+  font-size: 16px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:last-child {
+  margin-right: 65px;
+}
+
+header .logo {
+  display: inline-block;
+  width: 190px;
+  margin-right: -85px;
+
+}
+
+.billboard {
+	width: 880px;
+	height: 524px;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+}
+
+.billboard-text{
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	align-items: center;
+	font-size: 72px;
+ 	text-align: center;
+ 	margin-left: 135px;
+ 	margin-bottom: 10px;
+ 	width: 400px;
+}
+
+.billboard-text button{
+  background: white;
+  vertical-align: top;
+  padding: 8px 40px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.billboard .billboard-text button:hover{
+  background: black;
+  color: white;
+}
+
+.header {
+	margin-right: -112px;
+	margin-bottom: 10px;
+}
+
+.column-content {
+	border-top: 1px solid black;
+	border-bottom: 1px solid black;
+	padding: 39px 0px;
+	display: flex;
+	flex-direction: column;
+}
+
+.two-columns {
+	display: flex;
+	justify-content: space-between;
+}
+
+.two-columns .content {
+	width: 410px;
+}
+
+.middle-img {
+	margin: 45px 0px;
+}
+
+.three-columns {
+	display: flex;
+	justify-content: space-between;
+}
+
+.three-columns .content {
+	width: 275px;
+}
+
+.contact {
+	padding: 39px 10px 0px;
+
+}
+
+.contact p {
+	margin: 20px 0px;
+}
+
+footer {
+	margin: 32px 0px;
+	text-align: center;
+	font-size: 14px;
+}
+
+
+/*Services CSS*/
+.serv-header-img {
+  margin: 35px 0px; 
+}
+
+.serv-header {
+  padding-bottom: 45px; 
+}
+
+.serv-header h2 {
+  font-size: 28px;
+}
+
+.serv-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding-top: 40px;
+  border-top: 1px solid black;
+  border-bottom: 1px solid black;
+}
+
+.serv-block {
+  width: 420px;
+  border: 1px solid black;
+  margin-bottom: 40px;
+  background-color: whitesmoke;
+  padding: 15px;
+}
+
+.serv-block button {
+  margin-top: 15px;
+  background: white;
+  border: 1px solid black;
+  border-radius: 5px;
+  padding: 8px 35px;
+}
+
+.serv-facts {
+  display: flex;
+  padding-top: 40px;
+}
+
+.facts-container {
+  padding-right: 40px;
+}
+.facts-container ul{
+  margin-top: 15px;
+}
+
+.facts-container ul li {
+  margin: 10px 0;
+  list-style: square inside;
+}
+
+
+/* Desktop First */
+
+@media (max-width: 800px){
+	.container {
```

Comment:
These are pretty wide tabs - It looks to be almost 8 spaces??
It may be from the way of formatting.